README.md

```diff
@@ -192,3 +192,23 @@ Jenkins.getInstance().clouds.add(myCloud);
 Azure Container Agents Plugin collects usage data and sends it to Microsoft to help improve our products and services. Read our [privacy statement](http://go.microsoft.com/fwlink/?LinkId=521839) to learn more.
 
 You can turn off usage data collection in Manage Jenkins -> Configure System -> Azure -> Help make Azure Jenkins plugins better by sending anonymous usage statistics to Azure Application Insights.
+
+Changelog
+=========

```

Comment:
Please update the changelog in `CHANGELOG.md` in the project root directory. Thanks.